src/Microsoft.ML/Models/OneVersusAll.cs

```diff
@@ -13,10 +13,15 @@ namespace Microsoft.ML.Models
     public sealed partial class OneVersusAll
     {
         /// <summary>
-        /// Create OneVersusAll multiclass trainer.
+        /// One versus all learner (also in known as One vs the rest) is a multiclass learner with
+        /// strategy to fit one classifier per class. It picks each class and trains provided binary learner
```

Comment:
>binary learner [](start = 94, length = 14)

For binary learner, we seem to have settled on using the phrase "binary classifier" specifically.